packages/react-components/src/lib/components/NoticeBanner.tsx

```diff
@@ -0,0 +1,41 @@
+import { FiX } from 'react-icons/fi';
+import { GrAnnounce } from 'react-icons/gr';
+import { useState } from 'react';
+import { NetworkNotice, activeNotices } from '../utils/networkConfig';
+
+export const NoticeBanner = ({
+  notices,
+}: {
+  notices: Array<NetworkNotice>;
```

Comment:
Let's export the props as "NoticeBannerProps" like https://github.com/Agoric/ui-kit/blob/main/packages/react-components/src/lib/components/AmountInput.tsx#L9. I do this so that they're accessible from the typedocs like https://9418f1f2.ui-kit-dwm.pages.dev/types/_agoric_react_components.AmountInputProps